subsys/usb/host/usbh_shell.c

```diff
@@ -107,27 +107,14 @@ static int bazfoo_request(struct usbh_contex *const ctx,
 
 	shell_info(ctx_shell, "host: transfer finished %p, err %d", xfer, err);
 
-	while (!k_fifo_is_empty(&xfer->done)) {
-		struct net_buf *buf;
-
-		buf = net_buf_get(&xfer->done, K_NO_WAIT);
-		if (buf) {
-			/*
-			 * FIXME: We don not distinguish the context
-			 * of the request and always try to print it
-			 * as descriptor first. If it is not a known descriptor,
-			 * we show a hexdump in any case.
-			 * This is just simple enough for first steps and will
-			 * be revised with coming peripheral device management.
-			 */
```

Comment:
I think this comment should stay because it still holds.

Reply:
Actually it should be removed in the commit "usb: host: rework ch9 tools to work on USB device" not here.